### DISCUSSION TITLE
No module named 'pyserini.index.lucene' when index my own docs

### DISCUSSION THREAD

**Discussion by kakuibeyond** [castorini/pyserini#1138] [Q&A] [answered]
**desc:**
when I index my own docs in google colab by following the instructions below, I met `**No module named 'pyserini.index.lucene'**`，and I insert a new code block with ‘import pyserini.index.lucene’, the problem still exist.
**My steps are as follows:**
1. `!pip install pyserini==0.12.0`
2. create jsonl docs like [sample_collection_jsonl](https://github.com/castorini/pyserini/blob/master/tests/resources/sample_collection_jsonl),and move to 'drive/MyDrive/data/passage_head2000.json'
3. mount my google drive
4. `!python -m pyserini.index.lucene \
  --collection JsonCollection \
  --input drive/MyDrive/data/passage_head2000.json \
  --index indexes/p1000_index \
  --generator DefaultLuceneDocumentGenerator \
  --threads 1`

by the way, I have checked the source code in the repo, the dir 'pyserini/index/lucene/' do exist in this repo, so I really don't know why this error occurs, should I do something else?

**Comment by lintool** [comment, marked as answer]
Hi @kakuibeyond - use a newer version of pyserini, see https://github.com/castorini/pyserini/#release-history

We recently refactored the package structure.
